source/urban_flood_mitigation.rst

```diff
@@ -130,24 +130,32 @@ The following is a short description of each of the outputs from the urban flood
 
     * **serv_blt**: :math:`Service.built` values for this watershed (see equation :eq:`service.built`).  An indicator of the runoff retention service for the watershed.  Only calculated when the Built Infrastructure Vector input is provided.
 
-Appendix: Data sources and guidance for parameter selection
+Appendix: Data sources and Guidance for Parameter Selection
+
 ===========================================================
 
-The following table summarizes possible data sources for inputs specific to the urban flood risk mitigation model. Additional information on common InVEST inputs (e.g. LULC, evapotranspiration) can be found in the annual water yield model documentation.
-
-Table 1
-^^^^^^^
-.. csv-table::
-  :file: urban_flood_risk/urban_flood_mitigation_appendix.csv
-  :header-rows: 1
-  :name: Table 1
-
-Table 2
-^^^^^^^
-.. csv-table::
-  :file: urban_flood_risk/urban_flood_mitigation_soil_types.csv
-  :header-rows: 1
-  :name: Table 2
+:ref:`LULC <lulc>`
+^^^^^^^^^^^^^^^^^^
+
+:ref:`Watersheds <watersheds>`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:ref:`Design Storm Precipitation <design_storm>`

```

Comment:
One thing we lost here from the table is the connection between "Depth of Rainfall" as it's referred to in Data Needs, and "Design Storm Precipitation". Can we include "Depth of Rainfall" in this heading?

Reply:
Good point. Changed the heading to "Depth of Rainfall for Design Storm"